packages/auth/src/graphql/scalars/UInt8.ts

```diff
@@ -10,7 +10,7 @@ function coerceUInt8Value(value: number | string | unknown): number {
     throw new TypeError('Provided value is not number or string.')
   }
   const nValue = Number(value)
-  if (Number.isInteger(nValue) && value >= 0 && value <= 255) {
+  if (Number.isInteger(nValue) && nValue >= 0 && nValue <= 255) {
```

Comment:
(same change made in backend)

fixes new rule: Forbidden Implicit Coercions in Relational Operators

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-0.html#forbidden-implicit-coercions-in-relational-operators